fact_plots/utils.py

```diff
@@ -7,3 +7,14 @@ def merge_dicts(*dict_args):
     for dictionary in dict_args:
         result.update(dictionary)
     return result
+
+
+def kwarg_or_default(kwargs, kwarg_key, default):
```

Comment:
This implements `dict.get`, why not just use it?

Reply:
done